src/robot_dart/control/pd_control.cpp

```diff
@@ -26,9 +28,21 @@ namespace robot_dart {
             Eigen::VectorXd q = robot->positions(_controllable_dofs);
             Eigen::VectorXd dq = robot->velocities(_controllable_dofs);
 
+
+            Eigen::VectorXd error= Eigen::VectorXd::Zero(_control_dof);
+            for(int i=0;i<_control_dof;++i){
```

Comment:
Please adhere to the formatting standard of robot_dart (use the `.clang-format` from [here](https://github.com/resibots/format_code)).

Reply:
From now on I will be using that formatting style